mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -375,8 +401,7 @@ def _get_orderby_clauses(self, order_by_list: List[str], s: Search) -> Search:
                     sort_clauses.append({key: {'order': sort_order}})
         sort_clauses.append({"start_time": {'order': "desc"}})
```

Comment:
Probably for another review, but why do you always order by start_time desc ? In the UI we can chose to order by start time asc too

Reply:
<img width="736" alt="Capture d’écran 2020-09-18 à 17 59 32" src="https://user-images.githubusercontent.com/60222230/93619359-be595880-f9d8-11ea-9177-97864322a694.png">


It is the default order_by and the one use for tie breaking

Reply:
But if I want to order by asc start time it works